FuzzTesting/Sources/FuzzJSON/main.swift

```diff
@@ -8,7 +6,7 @@ public func FuzzJSON(_ start: UnsafeRawPointer, _ count: Int) -> CInt {
   var msg: Fuzz_Testing_Message?
   do {
     msg = try Fuzz_Testing_Message(
-      jsonUTF8Data: Data(bytes),
+      jsonUTF8Data: Array(bytes),
```

Comment:
Should this api also get renamed since it is no long a `Data` argument?

Reply:
Will change to `jsonUTF8Bytes`